client/src/pages/categorypage/CategoryPage.jsx

```diff
@@ -31,15 +16,14 @@ export default function CategoryPage() {
           Categories
         </button>
       </Link>
-      <h1 className="cTitle">{categoryFind.name.replaceAll("-", " ")}</h1>
+      <h1 className="cTitle">{currentCategory.name.replaceAll("-", " ")}</h1>
       <div className="categoryCards">
-        {result && result[0].id !== null ? (
-          result.map((v) => <VideoCard video={v} key={v.id} />)
+        {categoriesData && categoriesData[0].id !== null ? (
+          categoriesData.map((v) => <VideoCard video={v} key={v.id} />)
         ) : (
           <h2>There are no videos associated with this category</h2>
         )}
       </div>
-      <Filter category={categoriesData} />
```

Comment:
we originally added the filter component here to stick to the digital mock-up

Reply:
got it, will add it back

Reply:
fixed